logger.js

```diff
@@ -46,13 +47,20 @@ function pinoLogger (opts, stream) {
     }, 'request completed')
   }
 
+  function onReqAborted() {
+    this.res.statusCode = 0;
```

Comment:
@davidmarkclements how do you feel about overwriting the `statusCode` in here?
I wouldn't do it.